### ISSUE TITLE
Introduce avatars for staff accounts

### ISSUE THREAD

**Issue by maarcingebala** [mirumee/saleor#3862]
Staff users should be able to set their profile pictures in the Dashboard 2.0:

![image](https://user-images.githubusercontent.com/5421321/54752945-075fff00-4be0-11e9-94a0-3a6cf20ecfad.png)

Avatars would be displayed on various timelines (such as order history timeline) in the Dashboard 2.0. To make that possible, we need to first adjust the backend:

- [ ] add the avatar image field to the `User` model
- [ ] add a mutation to upload avatar image
- [ ] add a mutation to clear the avatar
- [ ] add a field to fetch the avatar when querying for the user's data

There are a few placeholders available on Figma that we can use in `populatedb` to generate example avatars.

This issue is related to backend only, for the Dashboard 2.0 we'll open a separate issue once backend is ready.

**Comment by jxltom** [comment]
I think this can be extended to that customers are also able to set their avatars in storefront 2.0 in later development?

**Comment by maarcingebala** [comment]
I rarely see avatars for customers in other e-commerce platforms. Since we have the same model for staff users and customers, it'll be easier to add in the storefront, but I guess we won't do that for now.

**Comment by jxltom** [comment]
That's true.

**Comment by pawelzar** [comment]
@maarcingebala ok, I'll take a look what we already have and I'll stick with the dark ones.
